From 66755a09d5128bd3d0df4016b31b4b6569fc4551 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Mon, 28 Sep 2015 12:06:38 -0700
Subject: [PATCH] Add rule to build intel_mid_pcihelpers.o

Add a makefile rule to build intel_mid_pcihelpers.o when
CONFIG_GMIN_INTEL_MID is enabled.
---
 arch/x86/platform/intel-mid/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/platform/intel-mid/Makefile b/arch/x86/platform/intel-mid/Makefile
index de29635..6a7979f 100644
--- a/arch/x86/platform/intel-mid/Makefile
+++ b/arch/x86/platform/intel-mid/Makefile
@@ -1,3 +1,4 @@
 obj-$(CONFIG_X86_INTEL_MID) += intel-mid.o
 obj-$(CONFIG_X86_INTEL_MID)	+= intel_mid_vrtc.o
 obj-$(CONFIG_EARLY_PRINTK_INTEL_MID)	+= early_printk_intel_mid.o
+obj-$(CONFIG_GMIN_INTEL_MID) += intel_mid_pcihelpers.o
-- 
1.7.9.5

